src/other/Table.js

```diff
@@ -235,14 +236,17 @@
         }
         var idx = this._columns.indexOf(column);
 
-        this._data.sort(function (l, r) {
-            if (l[idx] === r[idx]) {
-                return 0;
-            } else if (typeof (r[idx]) === "undefined" || l[idx] > r[idx]) {
-                return context._currentSortOrder;
-            }
-            return context._currentSortOrder * -1;
-        });
+        switch (this.sortType()) {
+            case "simple":
+                this._data.sort(function (a, b) {
+                    return context._currentSortOrder === 1 ? d3.ascending(a[idx], b[idx]) : d3.descending(a[idx], b[idx]);
```

Comment:
would be it faster (less iterations of that loop to not use Ternary Operator there)?